### DISCUSSION TITLE
Submit feature requests here!

### DISCUSSION THREAD

**Discussion by Vagr9K** [Vagr9K/gatsby-material-starter#58] [Ideas]
Since the starter is a more of a code boilerplate than a library, features generally get added during major Gatsby version updates to make sure that the current version users don't have a hard time merging bugfixes.

In this thread you can request new features to be added in the next major version!

**Comment by maddovr** [comment]
FontAwesome 5 or the possibility to ship with their own svg?

**Comment by Vagr9K** [comment]
After investigating for a while it seems like https://github.com/FortAwesome/react-fontawesome should be able to be integrated without any issues (add the dependency, import the component, done).

This project itself doesn't use FA5 icons, which means that this will be a redundant dependency if added out of the box. 
That being said, nothing prevents you from adding it manually later.

Sorry for the late reply. I've had a major lack of free time recently.

**Comment by daningamells** [comment]
Does this template have pagination out of the box?

**Comment by Vagr9K** [comment]
No.

There was an idea to use `gatsby-paginate`, but it was dropped due to licensing concerns related to GPLv3.
